index.js

```diff
@@ -69,3 +69,5 @@ export { default as ViewCustomFieldsSettings } from './lib/CustomFields/pages/Vi
 export { default as EditCustomFieldsSettings } from './lib/CustomFields/pages/EditCustomFieldsSettings';
 export { default as EditCustomFieldsRecord } from './lib/CustomFields/pages/EditCustomFieldsRecord';
 export { default as ViewCustomFieldsRecord } from './lib/CustomFields/pages/ViewCustomFieldsRecord';
+
+export * from './lib/utils';
```

Comment:
IMHO it's better to keep code style consistent - here it's hard to say what you are exporting actually...

Reply:
I proposed this change in fact.
For now it's like "just re-export whatever it is there".
Later we'll maybe have a whole bunch of API-related hooks using react-query, then we'll figure out the proper directory/module structure.